roles/config-hostname/README.md

```diff
@@ -0,0 +1,64 @@
+Role Description
+================
+
+This ansible role updates a hosts' hostname or FQDN
+
+Requirements
+------------
+
+No specific system requirements.
+
+Role Variables
+--------------
+
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+| hostname | The (short) hostname to use for the host | no | the 'inventory_hostname_short' fact |
+| dns_domain | DNS domain to append to the hostname to make a FQDN | no | |
+
+
+Dependencies
+------------
+
+N/A
+
+Example Playbooks
+----------------
+
+```
+- hosts: my-host
+  roles:
+  - config-hostname
+```
+
+Example Inventory
+----------------
+
+**inventory/hosts:**
+```
+[my-host]
+192.168.1.10
+```
+
+**inventory/group_vars/my-host.yml:**
+
+```
+---
+ansible_become: True
```

Comment:
LGTM. Only comment I have is around `ansible_become`. I know in other roles we've gone down the path of just using `become` on the necessary tasks. Did we want to do this here as well?

Reply:
Since all of these tasks requires elevated privileges, I think I could go either way. I can add a `block` to enforce it though - it's fine.